### DISCUSSION TITLE
Character wise diff

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [NeogitOrg/neogit#956] [Q&A]
Is there a way to get a more specific , character wise diff in the expanded staged/changed sections? 

> Foo Bar Baz
> Foo Bax Baz

**Comment by CKolkey** [comment]
It's in the works, but there's no ETA yet.

**Reply by LamprosPitsillos** [reply]
Great Thanks!
Is there an issue or branch to watch ?

**Reply by CKolkey** [reply]
Nope, not yet. I've only done some experiments so far

**Reply by LamprosPitsillos** [reply]
If you will be interested in creating a branch ,i would like to give it a look

**Reply by CKolkey** [reply]
I'll dig it up when I have the chance, though if I recall correctly there's not much yet. Mostly just read though how gitsingns handles this. I'll @ you when there's something :)
